From: Kan-Ru Chen <koster@debian.org>
Date: Sat, 18 Nov 2023 17:30:57 +0900
Subject: MuPDF crossbuild use host cc for utils

---
 Makerules | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Makerules b/Makerules
index e6df0a2..5df7aec 100644
--- a/Makerules
+++ b/Makerules
@@ -12,6 +12,8 @@ endif
 WARNING_CFLAGS := -Wall -Wsign-compare
 
 PKG_CONFIG ?= pkg-config
+C_FOR_BUILD ?= $(CC)
+HAVE_LIBDL ?= yes
 
 # Feature configuration options
 
