From: Kan-Ru Chen <koster@debian.org>
Date: Mon, 12 Feb 2024 10:31:51 +0900
Subject: define SONAME header

---
 Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/Makefile
+++ b/Makefile
@@ -48,6 +48,7 @@
 ifeq ($(LINUX_OR_OPENBSD),yes)
   ifneq ($(USE_SONAME),no)
     SO_VERSION = .$(VERSION_MINOR).$(VERSION_PATCH)
+    SONAME_LDFLAGS = -Wl,-soname,libmupdf.so$(SO_VERSION)
     ifeq ($(OS),Linux)
       SO_VERSION_LINUX := yes
     endif
@@ -104,7 +105,7 @@
 
 $(OUT)/%.$(SO)$(SO_VERSION):
 ifeq ($(SO_VERSION_LINUX),yes)
-	$(LINK_CMD) -Wl,-soname,$(notdir $@) $(LIB_LDFLAGS) $(THIRD_LIBS) $(LIBCRYPTO_LIBS)
+	$(LINK_CMD) $(SONAME_LDFLAGS) $(LIB_LDFLAGS) $(THIRD_LIBS) $(LIBCRYPTO_LIBS)
 	ln -sf $(notdir $@) $(patsubst %$(SO_VERSION), %, $@)
 else
 	$(LINK_CMD) $(LIB_LDFLAGS) $(THIRD_LIBS) $(LIBCRYPTO_LIBS)
